server/package.json

```
@@ -16,6 +16,7 @@
     "db:init": "tsx src/db/init.ts",
```

Biome (2.1.2)

[error] The key db:init was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)